Setup (#1334)
spryker/spryker-docs: docs/scos/dev/setup/installing-spryker-with-docker/installing-spryker-with-docker.md

## Installation

> There are several ways to install a Docker-based instance of Spryker. See [Choosing an installation mode](/docs/scos/dev/setup/installing-spryker-with-docker/installation-guides/choosing-an-installation-mode.html) to learn about them in detail.

Reviewer: ```suggestion
There are several ways to install a Docker-based instance of Spryker. For more details about these ways, see [Choosing an installation mode](/docs/scos/dev/setup/installing-spryker-with-docker/installation-guides/choosing-an-installation-mode.html).
```
